### ISSUE TITLE
Add option to choose party for specific bosses?

### ISSUE THREAD

**Issue by rdeee** [HughJeffner/FFRK-LabMem#158]
Augment the enemies list to set a preference for different parties to fight different bosses? Would provide an option for example to switch to a physical team from the main mage team against bosses with reflect.

**Comment by alextcl** [comment]
Suggestion of implemention

Extend EnemyPriority class
Add a properties Party Order as an array of number for target enemy

```json
"EnemyPriorityList":
[
{
 "Name": "Ultimate Weapon",
 "Enabled" : false,
 "PriorityAdjust": 1,
 "PartyOrder": [1,2,3]
}
]
```
Add another config value with 
```json
"PartyFatigueThreshold": 35
```

In method GetPartyIndex

1. Pass in the current combat enemy, and current fatigue info
2. look up the PartyOrder value based on enemy name
3. Select the first party in the array, if [1,2,3] then 1st party (also maybe set the first party as fail safe default)
4. If the party total fatigue is greater than threshold, switch to the following party in the array order
5. If all party fatigue exceed threshold, default to 1st party ond apply lether tears if enabled (may be return flag to use tears in the method)
6. Remove the party number from the array if a party should not be selected for the battle. E.g. to avoid party 2, then the array would be [1,3]

**Comment by HughJeffner** [comment]
Got requests to use a different party for the master painting.  The enemy list might not be the place for that though, since the same name can appear as a normal enemy.

**Comment by alextcl** [comment]
while the same name could be appeared in both normal and boss version, they share the same weakness chart. If your selected team can defeat the boss version, you should able to delete the normal version as well.

**Comment by HughJeffner** [comment]
I implemented this in 7.5 #176 

But I left out the fatigue setting for now.  The `Parties` property is a list, but only the first value is used.  An empty list falls back to default.

If there's a great need for filtering the party further based on fatigue, I'll add it as a new issue.
